### PR TITLE
AAP-2086 Add port 443 access prereq for syncing collections

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Automation Hub requires port 443 access for syncing certified collections. Add prereq for syncing procedure.
Affects `titles/hub/configuring-private-hub-rh-certified`

### CODE REVIEW

**Review by newswangerd — APPROVED** [review]

File: downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc [hunk 1]
```diff
@@ -12,6 +12,7 @@ To use only those collections specified by your organization, you must include a
 * You have valid *Modify Ansible repo content* permissions.
 See https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_user_access_in_private_automation_hub/index[Managing user access in Automation Hub] for more information on permissions.
 * You have retrieved the Sync URL and API Token from the {HubName} hosted service on {Console}.
+* You have configured access to port 443. This is required for syncing certified collections. For more informatiom, see the Automation hub table in the https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_planning_guide/ref-network-ports-protocols_planning[Network ports and protocols] chapter of the {PlatformName} Planning Guide.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc:15]: see the {HubName} table ?

Do we call it syncing? Don complained about that and said we should use the full  word.

**Review by ianf77 — COMMENTED** [review]
Couple of comments

File: downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc (hunk 1 above)
Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc]: Updated to use attribute and "synchronizing"

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**